packages/manager/modules/pci/src/projects/project/data-processing/data-processing.constants.js

```diff
@@ -52,3 +52,15 @@ export const METRICS_REFRESH_INTERVAL = 5000;
 
 export const JOB_TYPE_PYTHON = 'python';
 export const JOB_TYPE_JAVA = 'java';
+
+export const SUBMIT_JOB_API_GUIDES = {
+  DEFAULT: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  ASIA: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  AU: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  CA: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  FR: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  GB: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  IE: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  QC: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
+  SG: 'https://docs.ovh.com/gb/en/data-processing/use-api/',
```

Comment:
All the guides point to `gb/en`?

Reply:
Yes, so far it's the only doc we have but we'd rather plan ahead for when we do translate it.

Reply:
You can delete all except DEFAULT. It will have same behaviour. Add them when you have the proper links.